EthOn.rdf

```diff
@@ -1235,21 +1272,7 @@
         <ns:term_status>unstable</ns:term_status>
     </owl:DatatypeProperty>
 
-
-
-    <!-- http://ethon.consensys.net/msgGasPrice -->
-
-    <owl:DatatypeProperty rdf:about="http://ethon.consensys.net/msgGasPrice">
```

Comment:
No, `msgGasCost` was changed to `txGasCost`, but it appears I only made that change to master. Should be resolved by PR #123